packages/react-accordion/src/common/isConformant.ts

```diff
@@ -1,7 +1,9 @@
 import { isConformant as baseIsConformant, IsConformantOptions } from '@fluentui/react-conformance';
 
-export function isConformant(testInfo: Omit<IsConformantOptions, 'componentPath'>) {
-  const defaultOptions = {
+export function isConformant<TProps = {}>(
```

Comment:
@bsunderhus This change pointed out an existing issue with the AccordionPanel test.

It calls isConformant like this:
```ts
  isConformant({
    requiredProps: { open: true },
    Component: AccordionPanel,
    displayName: 'AccordionPanel',
  });
```

`open` is not included in `AccordionPanelProps`, and this now causes a TS error. It looks like `open` is in `AccordionPanelState` (and is apparently required to run the tests), but that's not the proper props interface to use for an `AccordionPanel`.  So I'm not sure what the right solution is here.

Reply:
This is actually right, and `{open: true} ` there is clearly a hack.

There should be no such property in `AccordionPanelProps`.  This `open` property is provided trhough Context by `AccordionItemContext.Provider`, but I'm not being able to mock it's behavior

At the moment the solution I'd propose is simply ignoring:

```tsx
isConformant({
    requiredProps: { open: true } as AccordionPanelProps,
    Component: AccordionPanel,
    displayName: 'AccordionPanel',
  });
```

Reply:
Updated, thanks!